AK/StdLibExtras.h

```diff
@@ -125,6 +125,17 @@ constexpr bool is_constant_evaluated()
 #endif
 }
 
+template <unsigned long size, unsigned long align>
+struct aligned_storage {
```

Comment:
Export this to the global namespace, `AK::Foo` isn't really common unless there's a name clash

Reply:
Can you change it to CamelCase, like all other Helper structs/classes/Concepts/constexprs-templates
also please move this struct definition to the front of the commit stack and inline the other changes

Reply:
I can squash the whole set if that helps? Would prefer to keep the set currently bisectable though, in case there is a functional change / break.

Reply:
I'm not in favor of squashing every fix into a single commit. But please move the `AlignedStorage` / `TypeAlignedStorage` definition to the front, and rebase the other changes on top of it.